share/ruby-install/package_manager.sh

```diff
@@ -38,8 +38,17 @@ function install_packages()
 		pkg)	$sudo pkg install -y "$@" || return $?     ;;
 		brew)
 			local brew_owner="$(/usr/bin/stat -f %Su "$(command -v brew)")"
-			sudo -u "$brew_owner" brew install "$@" ||
-			sudo -u "$brew_owner" brew upgrade "$@" || return $?
+			local installed=$(sudo -u "$brew_owner" brew list -1)
```

Comment:
Since casks are now shown with `brew list` this should probably have a `--formula` flag.